samples/cpp/ParallelLoopBody.cpp

```diff
@@ -0,0 +1,123 @@
+#include <opencv2/core/core.hpp>
+#include <opencv2/imgproc/imgproc.hpp>
+#include "opencv2/highgui/highgui.hpp"
+#include <iostream>
+
+using namespace cv;
+using namespace std;
+
+// This class use ParallelLoopBody to process image using thread
+// ParallelMeanLine process image to have for each line a mean value of 128 and standard deviation of 64
+class ParallelMeanLine: public cv::ParallelLoopBody
+{
+private:
+    cv::Mat &imgSrc;
+    vector<double> &meanLine;
+    vector<double> &stdLine;
+    bool verbose;
+
+public:
+    ParallelMeanLine(Mat& img, vector<double> &m,vector<double> &s):
+        imgSrc(img),
+        meanLine(m),
+        stdLine(s),
+        verbose(false)
+    {}
+    void Verbose(bool b){verbose=b;}
+    virtual void operator()(const Range& range) const
+    {
+
+        int h = imgSrc.cols;
```

Comment:
h is usually used to denote image height; probably, it should be renamed to w (for width) or ncols or something like that